### ISSUE TITLE
Display name for component is no longer correctly set.

### ISSUE THREAD

**Issue by delwaterman** [nkbt/react-copy-to-clipboard#63]
We get a failing snapshot test on our code related to the name of the CopyToClipboard class:

```
 -    <CopyToClipboard
 +    <Component
```

You need to set the displayName on the class:

```
CopyToClipboard.displayName = 'CopyToClipboard';
```

**Comment by nkbt** [comment]
I was waiting for babel to catch up, there is a pr, but it us still not merged.

Better solution would be switch to es6 classes and this problem should be gone.

**Comment by nkbt** [comment]
I am currently updating all my libs to use classes, so hold on, it is going to be fixed soon

**Comment by nkbt** [comment]
Published v5.0.0

https://github.com/nkbt/react-copy-to-clipboard/releases/tag/v5.0.0

Unless you are stuck on old version of React - update should be painless.
